### PR TITLE
VAGOV-2802: Open first level side nav on root level pages

### PR DESCRIPTION
**Pull request by acabouet**
## Description
On root level pages (ex. http://localhost:3001/outreach-and-events) have the first accordion in the side nav open by default. 

## Testing done
Locally with content from staging.cms.va.gov (no root level pages on dev) 

## Screenshots
Screen Shot 2019-06-04 at 1 11 18 AM

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by beeyayjay — APPROVED** [review]
LGTM!

File: src/applications/static-pages/sidenav-behavior.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
+/*
+First accordion item open by default
+*/
+
+export function openFirstLeftNavItem(sidenav) {
```
Comment (jbalboni) [inline src/applications/static-pages/sidenav-behavior.js]: Is it possible to do this at build time, instead? We have the url, it seems like we could avoid doing this client side and having that flash where the menu is closed, then open.

Comment (acabouet) [reply src/applications/static-pages/sidenav-behavior.js]: I'm down. Could you point me towards an example of integrating something like this into the build?

Comment (jbalboni) [reply src/applications/static-pages/sidenav-behavior.js]: I don't think we have anything exactly like this. The old sidebar template kind of does it, but it's not exactly the same. I think you'd need to update the sidebar templates to set `aria-expanded` and `aria-hidden` on the accordion buttons and sections depending on the loop index and `entityUrl.path`.

Comment (acabouet) [reply src/applications/static-pages/sidenav-behavior.js]: Was able to implement this in the sidebar templates.

**Review by jbalboni — APPROVED** [review]

**Pull request merged**